.github/workflows/release.yml

```
@@ -6,11 +6,11 @@ on:
 env:
   CARGO_TERM_COLOR: always
   IMAGE_NAME: plerkle-test-validator
-  RUST_VERSION: 1.69.0
-  SOLANA_VERSION_STABLE: v1.16.6
+  RUST_VERSION: 1.70.0
```

```suggestion
  RUST_VERSION: 1.73.0
```
should be 1.73.0
https://github.com/solana-labs/solana/blob/v1.17.20/rust-toolchain.toml#L2